src/main/groovy/org/gradle/android/Warnings.groovy

```diff
@@ -0,0 +1,23 @@
+package org.gradle.android
+
+import org.gradle.android.workarounds.CompileLibraryResourcesWorkaround_7_0
+
+import java.util.concurrent.atomic.AtomicBoolean
+
+enum Warnings {
```

Comment:
Using the enum is a clever singleton I just realized!
Does this mean it will be warned once per Gradle daemon or once per build?

Reply:
That's a good question.  Turns out it's once per daemon due to classloader caching.  Perhaps we should reset this at the end of each build session.